### DISCUSSION TITLE
experimental preview handlers gone

### DISCUSSION THREAD

**Discussion by ianamk** [sumatrapdfreader/sumatrapdf#3871] [General]
in the previews version of the premake5 file there wore defines to build a few experimental preview handlers 

        "BUILD_XPS_PREVIEW", "BUILD_DJVU_PREVIEW", "BUILD_EPUB_PREVIEW",
        "BUILD_FB2_PREVIEW", "BUILD_MOBI_PREVIEW", "BUILD_CBZ_PREVIEW",
        "BUILD_CBR_PREVIEW", "BUILD_CB7_PREVIEW", "BUILD_CBT_PREVIEW",
        "BUILD_TGA_PREVIEW"

in the current I premake5 I could only find

        "BUILD_EPUB_PREVIEW",
        "BUILD_FB2_PREVIEW", "BUILD_MOBI_PREVIEW"

Have the cbx djvu and xps handlers been removed for good, I'm guessing xps is handled internally by mupdf.
I know they wore experimental features never released in public builds but they did work well for years, any chance of them coming back I know mupdf for android natively supports cbx files are you using it to render commics or no, maybe the pdf handler can render cbx files thru mupdf

**Comment by kjk** [comment]
They are not gone. They are just built by default.
